Assets/Scripts/Enrico/Inventory.cs

```diff
@@ -1,66 +1,66 @@
-﻿using System.Collections;
-using System.IO;
-using System.Collections.Generic;
-using UnityEngine;
-using UnityEngine.UI;
-using PlayFab;
-using PlayFab.Json;
-using PlayFab.ClientModels;
-
-public class Inventory : MonoBehaviour
-{
-
-    //Instance Refs
-    private Manager manager;
-    Resources allResources;
-
-    public void GetCatalog()
-    {
-        manager = FindObjectOfType<Manager>().GetComponent<Manager>();
-        allResources = manager.allResources;
+﻿using System.Collections;
+using System.IO;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using PlayFab;
+using PlayFab.Json;
+using PlayFab.ClientModels;
+
+public class Inventory : MonoBehaviour
+{
+
+    //Instance Refs
+    private Manager manager;
+    Resources allResources;
+
+    public void GetCatalog()
+    {
+        manager = FindObjectOfType<Manager>().GetComponent<Manager>();
+        allResources = manager.allResources;
         foreach(var res in allResources.resourceInfo)
         {
             print(res.resourceName);

```

Comment:
Could easily be
`Each(allResources.resourceInfo, res => print(res.resourceName));`

Reply:
not mine